apps/api/src/app/subscribers-v2/subscribers.controller.ts

```diff
@@ -136,4 +140,24 @@ export class SubscribersController {
       })
     );
   }
+
+  @Get('/:subscriberId/preferences')
+  @UserAuthentication()
+  @ExternalApiAccessible()
+  @ApiOperation({
+    summary: 'Get subscriber preferences',
+    description: 'Get subscriber preferences',
+  })
+  async getSubscriberPreferences(
```

Comment:
Do we want also to define a custom sdk method name? I'm wondering how it will name it by default.

Reply:
I've named it `retrieve`, I think thats the convention